Gruntfile.js

```diff
@@ -223,6 +224,10 @@ var _              = require('lodash'),
                     src: ['core/test/unit/**/storage*_spec.js']
                 },
 
+                privateBlogging: {
+                    src: ['core/server/apps/private-blogging/tests/*_spec.js']
+                },
+
```

Comment:
Along the same lines as what halfdan suggested above--I'm not sure we need a distinct target for testing a specific app. I'd either drop this altogether, or make it generic, e.g., `apps-internal` or something.